metrics/web/test/debug_menu/presentation/state/debug_menu_notifier_test.dart

```diff
@@ -472,4 +473,4 @@ class _UpdateLocalConfigUseCaseMock extends Mock
 class _CloseLocalConfigStorageUseCaseMock extends Mock
     implements CloseLocalConfigStorageUseCase {}
 
-class _WebPlatformMock extends Mock implements WebPlatform {}
+class _RendererMock extends Mock implements Renderer {}
```

Comment:
Would it be great to extract this mock to the `test_utils` package to avoid duplication?